test/e2e/autoscaling/horizontal_pod_autoscaling.go

```diff
@@ -91,6 +91,18 @@ var _ = SIGDescribe("[Feature:HPA] Horizontal pod autoscaling (scale resource: C
 			scaleTest.run("rc-light", e2eautoscaling.KindRC, f)
 		})
 	})
+
+	ginkgo.Describe("[Serial] [Slow] ReplicaSet with idle sidecar (ContainerResource use case)", func() {
+		//ContainerResource CPU autoscaling
+		ginkgo.It("scale up on a busy application with an idle sidecar container "+titleUp, func() {
+			scaleOnIdleSideCar("rs", e2eautoscaling.KindReplicaSet, false, f)
+		})
+
+		//ContainerResource CPU autoscaling
+		ginkgo.It("should not scale up on a busy sidecar with an idle application", func() {
```

Comment:
nit to be consistent with the other tests
```suggestion
		ginkgo.It("Should not scale up on a busy sidecar with an idle application", func() {
```